legacysms/routers.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+import legacysms
+
+LOG = logging.getLogger(__name__)
+
+
+class StatsSchemaRouter:
+    """
+    This is a database router that routes queries for stats schema tables.
+    """
+
+    def db_for_read(self, model, **hints):
+        """
+        The only stats schema table defined is MediaStatsByDay,
+        so route queries on this to the stats DB.
+        """
+        if legacysms.models.MediaStatsByDay == model:
+            return 'stats'
+
+        return 'default'
+
+    def db_for_write(self, model, **hints):
+        """
+        The only stats schema table defined is MediaStatsByDay,
+        so route queries on this to the stats DB.
+        """
+        if legacysms.models.MediaStatsByDay == model:
+            return 'stats'
```

Comment:
Since ``db_for_{read,write}`` are identical, could we replace them both with a call to a single private method?